infoblox_client/utils.py

```diff
@@ -345,3 +345,16 @@ def safe_json_load(data):
         return jsonutils.loads(data)
     except ValueError:
         LOG.warn("Could not decode reply into json: %s", data)
+
+
+def try_value_to_bool(value):
+    """Tries to convert value into boolean.
+
+    If value is not a string representation of True or False,
+    then unchanged incoming value is returned.
+    """
+    if value == 'True':
+        return True
+    elif value == 'False':
```

Comment:
I think this should be case insensitive instead. In Infoblox-netmri I did case insensitive true, on, yes maps to True and false, off, no to False. No one needs to name a file 'Off' anyway.